ante/tests/fee_test.go

```diff
@@ -0,0 +1,393 @@
+package ante_tests
+
+import (
+	cheqdante "github.com/cheqd/cheqd-node/ante"
+	cheqdpost "github.com/cheqd/cheqd-node/post"
+	didtypes "github.com/cheqd/cheqd-node/x/did/types"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	"github.com/cosmos/cosmos-sdk/testutil/testdata"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/bank/testutil"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Fee tests", func() {
+	s := new(AnteTestSuite)
+
+	It("Test Ensure Zero Mempool Fees On Simulation CheckTx", func() {
```

Comment:
@DaevMithran Could you please remove `Test...` as a prefix?
It's not necessary in BDD descriptions in Ginkgo.
As for `testify` it's required to distinguish procedural test cases.

Same goes for all occurrences.

Reply:
Fixed this